app/javascript/app/utils/utils.js

```diff
@@ -10,6 +10,26 @@ export function isCountryIncluded(countriesIncluded = [], iso) {
   return countriesIncluded.includes(iso);
 }
 
+export function compareIndex(a, b) {
```

Comment:
It would be great if we could make this more reusable for different data structures. Simply be passing in the data key you want to access the sdg number on (i.e. 'value' or 'number') and then comparing and returning the sorted list. This would mean we can reuse in other parts of the site where we aren't necessarily using the targets data structure.

Reply:
Good idea!

Reply:
Something like lodash/sortBy would be great.